driver.go

```diff
@@ -83,9 +85,9 @@ func init() {
 	if !skipRegisteration() {
```

Comment:
do we want to fix this typo as well perhaps? :)

Reply:
Well caught! So I fixed the function name, but the same typo exists in the env var, which I can't fix :(